pkg/search/analytics/getanalytics.go

```diff
@@ -0,0 +1,132 @@
+package analytics
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"time"
+
+	"github.com/algolia/algoliasearch-client-go/v3/algolia/search"
+	"github.com/mark3labs/mcp-go/mcp"
+	"github.com/mark3labs/mcp-go/server"
+
+	"github.com/algolia/mcp/pkg/mcputil"
+)
+
+const (
+	analyticsBaseURL = "https://analytics.algolia.com/2"
```

Comment:
This won't work for applications with their log region set in Germany 👀

Reply:
Indeed, it routes to the closest location based on the documentation:
- https://analytics.algolia.com (routes requests to the closest of the above servers, based on your geographical location)

I will add the log region in the configuration